ethcore/sync/src/chain/mod.rs

```diff
@@ -1253,6 +1263,9 @@ impl ChainSync {
 		packet.append(&chain.total_difficulty);
 		packet.append(&chain.best_block_hash);
 		packet.append(&chain.genesis_hash);
+		if ethereum_protocol_version >= ETH_PROTOCOL_VERSION_64.0 { 
+			packet.append(&self.fork_filter.current(io.chain()));
```

Comment:
this will break `warp_protocol` packet with eth63, right?

Reply:
I'm afraid it will be. Should I bump parity protocol to version 4 then?

Reply:
Actually it's fine as long as we can correctly identify both `eth_protocol_version` and `warp_protocol_version` per peer.